builder/codebuild/codebuild.go

```diff
@@ -182,21 +181,29 @@ func (b *Builder) build(ctx context.Context, w io.Writer, opts builder.BuildOpti
 
 	sess := session.Must(session.NewSession())
 
-	r, err := cloudwatch.NewLogger(sess, logInfo.GroupName).Open(logInfo.StreamName)
+	r, err := cloudwatch.NewGroup(logInfo.GroupName, cloudwatchlogs.New(sess)).Open(logInfo.StreamName)
 
 	if err != nil {
 		return err
 	}
 
+	copyDone := make(chan bool)
+
 	go func() {
 		io.Copy(w, r)
```

Comment:
[io.Copy](https://golang.org/pkg/io/#Copy) returns `(int64, error)`. We should probably make `copyDone` a `chan error` so we can return any errors from trying to Copy the logs:

```golang
_, err := io.Copy(w, r)
copyDone <- err
```